modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavaMSF4JServerCodegen.java

```diff
@@ -0,0 +1,152 @@
+package io.swagger.codegen.languages;
+
+import io.swagger.codegen.*;
+import io.swagger.models.Operation;
+
+import java.util.*;
+
+import org.apache.commons.lang3.BooleanUtils;
+import org.apache.commons.lang3.StringUtils;
+
+public class JavaMSF4JServerCodegen extends AbstractJavaJAXRSServerCodegen {
+
+    protected static final String LIBRARY_JERSEY1 = "jersey1";
+    protected static final String LIBRARY_JERSEY2 = "jersey2";
+
+
+    public static final String DEFAULT_LIBRARY = LIBRARY_JERSEY2;
+
+    public JavaMSF4JServerCodegen() {
+        super();
+        outputFolder = "generated-code/JavaJaxRS-Jersey";
```

Comment:
Actually I found some other Java server stub generators do not have proper outputFolder set.

I'll file a PR to clean those up (including this one)